.gitignore

```diff
@@ -2,3 +2,8 @@
 .idea/tasks.xml
 .idea/workspace.xml
 target
+bin
```

Comment:
I'm no expert in .gitignore + Eclipse + Windows, but the internet seems to think that this should be `bin/` (or possibly `/bin/` or `bin/**`). Can you confirm this is correct?

Reply:
I just followed the entry you have on line 4. With it you're ignoring all "target" directories by just specifying `target`. That seems to work fine and `bin` works as well - I tested it.

But you're right - `bin/` is the recommended way, so I'll update the pull request. I hope you won't mind modifying the `target` entry to follow suit as well.